ur_simulation_gz/doc/installation.rst

```diff
@@ -0,0 +1,47 @@
+Installation
+============
+
+Skip any of below steps is not applicable.
+
+Setup ROS2 Workspace
+--------------------
+
+1. Create a colcon workspace:
+
+.. code-block:: console
+
+   $ export COLCON_WS=~/workspaces/ur_gz
+   $ mkdir -p $COLCON_WS/src
+
+.. note::
+   Feel free to change `~/workspaces/ur_gz` to whatever absolute path you want.
```

Comment:
Classic "I am used to markdown"

```suggestion
   Feel free to change ``~/workspaces/ur_gz`` to whatever absolute path you want.
```

Reply:
I like markdown very much.
Thanks for the fix!